Sources/Commands/SwiftPackageResolveTool.swift

```diff
@@ -8,32 +8,41 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
+import Basic
 import PackageGraph
 import SourceControl
 
-private class ResolverToolDelegate: DependencyResolverDelegate {
+private class ResolverToolDelegate: DependencyResolverDelegate, CheckoutManagerDelegate {
     typealias Identifier = RepositoryPackageContainer.Identifier
 
     func added(container identifier: Identifier) {
         print("note: considering repository: \(identifier.url)")
     }
+
+    func fetching(handle: CheckoutManager.RepositoryHandle, to path: AbsolutePath) {
+        print("note: fetching \(handle.repository.url) to \(path.asString)")
```

Comment:
I know that this is WIP, but this will eventually go through some kind of status reporting API, right?